src/python/bot/fuzzers/libFuzzer/peach/pits.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Dictionary to keep track of pit information."""
+
+from metrics import logs
+from system import environment
+import os
+
+
+def get_path(grammar):
+  """Return the path of the peach pit for the given grammar. Return None if the
+  Pit does not exist or the grammar is None."""
```

Comment:
do you anticipate it ever being called with an empty `grammar`? I'd rather make sure on the caller side that the grammar is always non-empty and then log an error here if it ever gets call with an empty argument. Otherwise, all these cases will go silent and may hide some other issue from us